src/components/ui/Hero/hero.scss

```diff
@@ -0,0 +1,93 @@
+$media: 1280px;
+
+[data-store-hero] {
+  display: flex;
+  width: 100%;
+  background-color: var(--bg-secondary-default);
+
+  @media screen and (max-width: $media) {
```

Comment:
Not sure we have some guidelines about this or if it is a big deal, but it looks like we're starting with desktop styles and using media queries to style for mobile. I'd vote for a [mobile-first approach](https://developer.mozilla.org/en-US/docs/Web/Progressive_web_apps/Responsive/Mobile_first) instead.

Reply:
You're right, i'll put in mobile first